styles/style.css

```diff
@@ -0,0 +1,358 @@
+* {
+  box-sizing: border-box;
+  margin: auto;
+}
+main {
+  padding: 0.5rem 9.5rem 4rem 9.5rem;
```

Comment:
this padding might not be needed. I'd rather have a max width on this container and center it with margin: 0 auto;  at smaller widths it looks a little strange